feature-libs/asm/components/customer-list/customer-list.component.ts

```diff
@@ -161,6 +172,17 @@ export class CustomerListComponent implements OnInit, OnDestroy {
 
       this.asmCustomerListFacade.customerListCustomersSearch(options);
     }
+    this.customerListConfig?.columns?.find((item) => {
```

Comment:
why use find?  the input of the find should be a filter like '(i > 0)'

Reply:
`find` is used  here because it's commonly used by spa like this. Do you have any other suggestion?

Reply:
See above

Reply:
other columns will be added later. We had better not use fixed index